modules/admin_manual/pages/release_notes.adoc

```diff
@@ -60,6 +60,8 @@ Password reset links sent to a user were invalid, if the user attempted to login
 Removing a user from a group using the user management UI resulted in an error that required the page to be refreshed to show the changes. This has been corrected.
 * **Added -y option to `occ encryption:encrypt-all` command** +
 The occ command `encryption:encrypt-all` now offers a `-y` option that can be used to automatically answer potential questions with "yes" which is particularly important for automated deployments with Ansible or similar tools.
+* **Fixed an issue with loading JS files when multiple apps folders are in use** +
+Previously ownCloud would have taken the files from the `apps/` folder even though there might be newer versions in e.g. `apps-external`. This has been changed for that ownCloud will always take the files from the most recent app version.
```

Comment:
```suggestion
Previously ownCloud would have taken the files from the `apps/` folder even though there might be newer versions in e.g. `apps-external`. This has been changed so that ownCloud will always take the files from the most recent app version.
```

Reply:
thx